Osteppy/src/server.js

```diff
@@ -1,32 +1,97 @@
 #!/usr/bin/env node
 
-/***********************************************************
+/** ********************************************************
 // OSTEP Dashboard Osteppy API
 // server.cpp
 // Date Created: 2018/09/11
 // Author: Olga Belavina and Yiran Zhu
 // Email: yzhu132@myseneca.ca
-// Description: Slack API triggered by and responds to 
-// slash commands such as /eod and /eod_left. 
-***********************************************************/
+// Description: Slack API triggered by and responds to
+// slash commands such as /eod and /eod_left.
+********************************************************* */
 
+import cp from 'child_process';
 import http from 'http';
+import path from 'path';
 import express from 'express';
 import cors from 'cors';
 import morgan from 'morgan';
 import bodyParser from 'body-parser';
-import config from './config.json';
 import axios from 'axios';
-import * as fs from 'fs';
+import fs from 'fs';
+import config from './config.json';
+
+class EODList {
+  constructor({
+    dataFile = path.join(__dirname, 'eods.json'),
+    unsubmittedNamesFile = path.join(__dirname, 'sleepyRAs.txt'),
```

Comment:
Not really sure why we are using different file formats (maybe @naiuhz knows), I think it would make sense to use `.json` only (parsing `.txt` is more error prone)

Reply:
`sleepyRA.txt` seems to just be a newline-delimited file so it's not hard to parse, but we could definitely transition everything to JSON.

Reply:
Changing the filetype is also going to require modifying `EODreminder.js`, so it might be better to hold off and change the filetype everywhere as a separate PR.

Reply:
okc :+1: